wrapper/pre_copy.py

```diff
@@ -698,6 +715,36 @@ def _get_nbdkit_cmd(self, disk, filters, final):
 
         return nbdkit_cmd
 
+    def _get_nbdkit_cmd_ssh(self, disk, filters, final):
+        print("*** %s" % disk.absolute_path.replace('.vmdk', '-flat.vmdk'))
+        nbdkit_cmd = [
+            'nbdkit',
+            '-v',
+            '-U', disk.sock,
+            '-P', disk.pidfile,
+            '--exit-with-parent',
+            '--readonly',
+            '--foreground',
+            '--exportname=/',
+        ] + [
+            '--filter=' + f for f in filters
+        ] + [
+            '--filter=log',
```

Comment:
Most of this (and some stuff below) should be similar in both functions, no need to duplicate the code, just add it to something like `_get_nbdkit_cmd_common()` or something and call it from both.